### PR TITLE
feat: wfinfo data services

### PR DESCRIPTION
**Pull request by TobiTenno**
### What did you fix? 
create cached data urls for [WF Info](https://wfinfo.warframestat.us)

---

### Reproduction steps

---

### Considerations
- Does this contain a new dependency? **[Yes]**
- Does this introduce opinionated data formatting or manual data entry? **[No]**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **[No]**
- Have I run the linter? **[No]**
- Is is a bug fix, feature request, or enhancement? **[Feature]**

### CODE REVIEW

File: src/controllers/wfinfo.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}));
+
+router.get('/prices', cache('1 hour'), ah(async (req, res) => {
+  logger.silly(`Got ${req.originalUrl}`);
+
+  if (Settings.wfInfo?.prices) {
+    return res.status(200).json(infoCache.getKey('prices'));
+  }
+  return res.status(503).json({ code: 503, error: 'WFInfo Data Services Unavailable' });
+}));
```
Comment (dimon222) [inline src/controllers/wfinfo.js:31]:
```suggestion
router.get('/filtered_items', cache('1 hour'), ah(async (req, res) => {
  logger.silly(`Got ${req.originalUrl}`);

  if (Settings.wfInfo?.filteredItems) {
    var infoCache = flatCache.load('.wfinfo', path.resolve(__dirname, '../../'));
    return res.status(200).json(infoCache.getKey('filteredItems'));
  }
  return res.status(503).json({ code: 503, error: 'WFInfo Data Services Unavailable' });
}));

router.get('/prices', cache('1 hour'), ah(async (req, res) => {
  logger.silly(`Got ${req.originalUrl}`);

  if (Settings.wfInfo?.prices) {
    var infoCache = flatCache.load('.wfinfo', path.resolve(__dirname, '../../'));
    return res.status(200).json(infoCache.getKey('prices'));
  }
  return res.status(503).json({ code: 503, error: 'WFInfo Data Services Unavailable' });
}));
```

Comment (TobiTenno) [reply src/controllers/wfinfo.js]: fixed in 08fadbf

**Review by dimon222 — APPROVED** [review]
LGTM

**Pull request merged**